### PR TITLE
US133021 Delete Schedule

### PR DESCRIPTION
**Pull request by IhabAlkabary**
image

image

https://rally1.rallydev.com/#/431372673576d/iterationstatus?detail=%2Fuserstory%2F612474758355

### CODE REVIEW

**Comment by IhabAlkabary** [comment]
Which one is the main one?
image

**Comment by Syeda-Azim** [comment]
> Which one is the main one? For now I duplicated the lang terms
> 
> image

en-us.js is the one, but we also write the same lang-term to en.js

File: src/components/module-scheduler-manager.js [hunk 1]
```diff
@@ -116,6 +116,25 @@ class ModuleSchedulerManager extends BaseMixin(LocalizeMixin(LitElement)) {
 		this.openDialog = false;
 	}
 
+	async _handleDelete() {
+		this.allSchedules.find(schedule => schedule.scheduleId === this._scheduleId);
+
+		await this.scheduleService.deleteSchedule(this._scheduleId);
+
+		this.requestUpdate();
+		this.openDialog = false;
```
Comment (Syeda-Azim) [inline src/components/module-scheduler-manager.js:125]: `this.openDialog` variable is used to handle both the ApplyNow and Delete dialog.

Comment (IhabAlkabary) [reply src/components/module-scheduler-manager.js:125]: they can't show at the same time, so why not

Comment (IhabAlkabary) [reply src/components/module-scheduler-manager.js:125]: you are right, I just checked and there could be interference. I'll fix.

**Review by Syeda-Azim — APPROVED** [review]

**Pull request merged**